src/pages/resources/openapi.json

```diff
@@ -7229,15 +7229,18 @@
                 "text",
                 "tables"
               ]
-            }
+            },
+           "default": [
+             "text"
+            ]
           },
           "tableOutputFormat": {
             "description": "Specifies the format of table contents output. Possible values are csv and xlsx.",
             "type": "string",
             "default": "xlsx"
           },
           "renditionsToExtract": {
-            "description": "List of elements whose renditions should be generated. Possible values are figures and tables.",
+            "description": "List of elements whose renditions should be generated. Possible values are figures and tables. If elementsToExtract parameter holds the value 'tables', it is mandatory for renditionsToExtract parameter to hold the same value or else it will fail.",
```

Comment:
I'd add "the" in front of elementsToExtract parameter.

Reply:
Okay I will add it.